server/tests/e2e/decentralized.spec.ts

```diff
@@ -114,4 +119,148 @@ describe('end-to-end decentralized', function () {
   it('several rounds of cifar 10 with three secure aggregators yields consensus', async () => {
     await reachConsensus('secure', 3)
   })
+
+  it("peers emit expected statuses", async function () {
+    this.timeout(15_000);
+    const lusCovidTask = defaultTasks.lusCovid.getTask();
+    lusCovidTask.trainingInformation.aggregationStrategy = 'mean';
+    lusCovidTask.trainingInformation.epochs = 8;
+    lusCovidTask.trainingInformation.roundDuration = 2;
+    lusCovidTask.trainingInformation.minNbOfParticipants = 2;
```

Comment:
less character, mr rdbl

```suggestion
    const lusCovidTask = {
      ...defaultTasks.lusCovid.getTask(),
      aggregationStrategy: "mean",
      epochs: 8,
      roundDuration: 2,
      minNbOfParticipants: 2,
    };
```

Reply:
I think it should be something like:
```js
const lusCovidTask = {
      ...defaultTasks.lusCovid.getTask(),
      trainingInformation: {
        ...defaultTasks.lusCovid.getTask().trainingInformation,
        scheme: "decentralized",
        aggregationStrategy: "mean",
        epochs: 8,
        roundDuration: 2,
        minNbOfParticipants: 2,
      } as TrainingInformation
    };
```
I find this harder to read than setting parameters one by one but maybe the nested spread can be simplified.

Reply:
Actually I find this one better:
```js
const lusCovidTask = defaultTasks.lusCovid.getTask();
lusCovidTask.trainingInformation = {
  ...lusCovidTask.trainingInformation,
  scheme: 'decentralized',
  aggregationStrategy: 'mean',
  epochs: 8,
  minNbOfParticipants: 2,
}
```